Cargo.toml

```diff
@@ -36,6 +36,7 @@ protobuf-codec = [
   "raft/protobuf-codec",
   "sst_importer/protobuf-codec",
 ]
+s3 = ["external_storage/s3"]
```

Comment:
Shall we add this in Makefile?

Reply:
I removed this feature for now. Seems to me there isn't cases where would want to disable it in code?